spring-cloud-stream-app-descriptor/src/main/resources/1.0.0.BUILD-SNAPSHOT/kafka-apps-docker.properties

```diff
@@ -0,0 +1,46 @@
+source.file=docker://org.springframework.cloud.stream.app:file-source-kafka:@project.version@
```

Comment:
How does the file source or sink work in a docker environment? I'd just leave them out.